src/uct/ib/base/ib_iface.h

```diff
@@ -81,6 +81,12 @@ enum {
     UCT_IB_ADDRESS_PACK_FLAG_PKEY          = UCS_BIT(5)
 };
 
+enum {
+    UCT_IB_IFACE_MLX5_RX_COMPLETION_ZIPPED,
+    UCT_IB_IFACE_MLX5_TX_COMPLETION_ZIPPED,
+    UCT_IB_IFACE_MLX5_CQE_ZIPPING_QUEUES_CNT,
```

Comment:
I understand that it is used by MLX5-only code and we don't have special `uct_ib_mlx5_iface_t`, but ...
wdyt about removing `MLX5` from their names?

Reply:
also, they should have `STAT` after `IFACE` in their names, i.e.
```suggestion
    UCT_IB_IFACE_STAT_RX_COMPLETION_ZIPPED,
    UCT_IB_IFACE_STAT_TX_COMPLETION_ZIPPED
    UCT_IB_IFACE_STAT_CQE_ZIPPING_QUEUES_CNT
```

Reply:
Yes, I think it is a good idea.